Main/FileListView.cpp

```diff
@@ -22,16 +23,27 @@
 #include "FileListItem.h"
 #include "PecoApp.h"
 
+FileListRow::FileListRow(FileListItem* item) : BRow(), fItem(item)
+{
+	int32 i = 0;
+	SetField(new BBitmapField(item->fIcon), i++);
+	SetField(new BStringField(item->fListName), i++);
+	SetField(new BStringField(item->fListGroesse), i++);
+	SetField(new BStringField(item->fListZeit), i++);
+	SetField(new BStringField(item->fListNewName), i++);
+}
+
 FileListView::FileListView(BRect frame)
-	: BListView(frame, "fileListView", B_SINGLE_SELECTION_LIST, B_FOLLOW_ALL, B_FRAME_EVENTS|B_WILL_DRAW|B_NAVIGABLE) {
+	: BColumnListView(frame, "fileListView", B_FOLLOW_ALL, B_FRAME_EVENTS|B_WILL_DRAW|B_NAVIGABLE, B_NO_BORDER, false) {
 }
 
 void FileListView::MouseDown(BPoint where) {
-	BListView::MouseDown(where);
+	BColumnListView::MouseDown(where);
 //	DeselectAll();
 };
 
-bool FileListView::InitiateDrag(BPoint where, int32 itemIndex, bool initialySelected) {
+bool FileListView::InitiateDrag(BPoint where, bool initialySelected) {
+	int32	itemIndex = IndexOf(RowAt(where));
 	FileListItem	*Item = (FileListItem *)ItemAt(itemIndex);
```

Comment:
This is superfluous, you can get the item first (using RowAt), then the index (using IndexOf), and avoid the extra ItemAt call.